hathor/nanocontracts/runner/types.py

```diff
@@ -76,6 +77,7 @@ class SyscallUpdateTokensRecord:
     htr_amount: int
     token_symbol: str | None = None
     token_name: str | None = None
+    token_version: TokenInfoVersion = TokenInfoVersion.DEPOSIT
```

Comment:
We should use `None` as default here, otherwise mint and melt operations might have an inconsistent token version.